### PR TITLE
docs: Add installation steps for helm charts

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Add documentation related to our official helm charts!

### CODE REVIEW

**Review by alban — APPROVED** [review]
Thanks!

I would also like to have a section "Using ig without installation" and referring to the [kubectl debug node](https://inspektor-gadget.io/docs/v0.28.1/ig/#using-ig-with-kubectl-debug-node) documentation.

**Comment by mqasimsarfraz** [comment]
> I would also like to have a section "Using ig without installation" and referring to the [kubectl debug node](https://inspektor-gadget.io/docs/v0.28.1/ig/#using-ig-with-kubectl-debug-node) documentation.

Good idea! I included it a separate commit!

File: docs/getting-started/install-kubernetes.md [hunk 1]
```diff
@@ -17,13 +17,15 @@ description: >
...
   - [Deploying with an AppArmor profile](#deploying-with-an-apparmor-profile)
+  - [Helm Chart Installation](#helm-chart-installation)
   - [Specific Information for Different Platforms](#specific-information-for-different-platforms)
     - [Minikube](#minikube)
 - [Uninstalling from the cluster](#uninstalling-from-the-cluster)
 - [Version skew policy](#version-skew-policy)
 - [Installing `ig`](#installing-ig)
   - [Install a specific release](#install-a-specific-release-1)
   - [Compile from source](#compile-from-source-1)
+  - [Using ig without installation](#using-ig-without-installation)
```
Comment (alban) [inline docs/getting-started/install-kubernetes.md:28]: I wonder if it should be in its own top section and cover both running ig without installation on 1. Linux hosts and 2. Kubernetes clusters (and just link to the appropriate part of ig.md).

Comment (mqasimsarfraz) [reply docs/getting-started/install-kubernetes.md]: Agreed, it looked bit off here. I added a new section "Quick Start" to make it clear. Please feel free to have another look! :)

**Review by alban — APPROVED** [review]
Thanks!

**Comment by mqasimsarfraz** [comment]
Thanks for the review!

**Pull request merged**